scripts/ci/_utils.sh

```diff
@@ -38,6 +38,9 @@ function verbose_docker {
 }
 
 function initialize_breeze_environment {
+    # default python version
+    PYTHON_VERSION=${PYTHON_VERSION:="3.6"}
```

Comment:
Should this go under `function setup_default_breeze_variables` ?

Reply:
Not really. The name here is misleading -  setup_default_breeze_variables is for breeze command. This one above initializes common environment that is used by both breeze and the CI scripts (CI scripts do no use Breeze as such - they share common methods with it). I will change the name to initialize_common_environment in a separate commit :)

Reply:
👍 Ya renaming it would be good, thanks

Reply:
https://github.com/apache/airflow/pull/7805/files